test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/DisposeWithoutEventUnsubscribedTest.cs

```diff
@@ -27,7 +28,7 @@ public class DisposeWithoutEventUnsubscribedTestAndAnimationDisposeTest : IExamp
         public void Activate()
         {
             DisposeWithoutUnsubscribedEventTest();
-            AnimationDisposeTest();
+            //AnimationDisposeTest();
```

Comment:
Thanks for adding an updated example.
BTW, why do you comment at this line and 296 line? Probably, for history?

Reply:
good point~ 
it is difference purpose so I added more comments to remind easily.
thank you.

Reply:
You updated the comment! Thanks for the feedback. :)
It's really good at scenario.